usagov/vote-gov-drupal
# Vote-1441 -> In English string updates

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-1441](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1441&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This PR will add an `in English` label to links that will lead a user to external URLs that are not in the native translation.

## Deployment and testing (required, if applicable)

### Post-deploy steps
1. Update About Vote.gov links on the page for the following translations ->  bn, zh-hans, zh, ht(2nd link), hi, km, ko, tl, vi, ypk
2. Update on Russian for email sign-up block

### Testing steps
1. pull down branch 
2. start local 
3. run `lando import-translations` 
4. run `lando drush uli` to log in as a superuser so that you can see all translations. 
5. check the home page and use the screen shot below to refer which stings needs the `in English added to them

homepage:
![homepage](https://github.com/usagov/vote-gov-drupal/assets/88721460/72d1f422-22ed-4425-b4cd-94d567e0e714)
